akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1150,8 +1149,10 @@ private[akka] class DDataShardCoordinator(
     case additional   => WriteMajorityPlus(settings.tuningParameters.waitingForStateTimeout, majorityMinCap, additional)
   }
 
-  implicit val node = Cluster(context.system)
-  private implicit val selfUniqueAddress = SelfUniqueAddress(node.selfUniqueAddress)
+  // FIXME make the all shards consistency is reflected in the ddata remembered thingy
```

Comment:
So we deal with the consistency improvements for the coordinator separately?

Reply:
They'd already been ported but i wasn't sure when merging which sets of consistencies were remembered entities (majority) vs the new ones (all or majority plus).

Patrik had ported that PR to this branch so you're right there aren't many changes